app-src/player-router/Makefile

```diff
@@ -0,0 +1,18 @@
+BROWSERIFYCMD = node_modules/.bin/browserify -d
+
+SMOKECHROME = node_modules/.bin/tap-closer | \
+	node_modules/.bin/smokestack -b chrome
+
+SMOKEFIREFOX = node_modules/.bin/tap-closer | \
+	node_modules/.bin/smokestack -b firefox
+
+test: test-chrome test-firefox
+
+test-chrome:
+	$(BROWSERIFYCMD) tests/basictests.js | $(SMOKECHROME)
```

Comment:
This Makefile should be deleted, which I'll do shortly, since the targets here have been moved into the main Makefile. In the main Makefile, I'll add -d so that source maps get built for the test runs. (Here, the -d is part of the `BROWSERIFYCMD` target.)